doc/user/content/guides/postgres-cloud.md

```diff
@@ -99,9 +99,7 @@ As a user with the `cloudsqlsuperuser` role, make these changes to the upstream
 
 1. In the Google Cloud Console, set the `cloudsql.logical_decoding` to `on`. This enables logical replication.
 
-1. The Materialize replica will need access to connect to the upstream database. In the Google Cloud Console, enable access on the upstream database for the Materialize replica's IP address.
-
-      If you are using Materialize Cloud, you will need to make the Cloud SQL instance publically accessible; Materialize Cloud instances do not have static IP addresses.
+1. The Materialize replica will need access to connect to the upstream database. In the Google Cloud Console, enable access e upstream database for the [Materialize instance's static IP address](/cloud/security/#static-ip-addresses)
```

Comment:
```suggestion
1. The Materialize replica will need access to connect to the upstream database. In the Google Cloud Console, enable access on the upstream database for the Materialize replica's IP address.

      If you are using Materialize Cloud, you can follow [these steps](/cloud/security/#static-ip-addresses) to get the static IP address of your instance.
```

Reply:
Not sure if you omitted the initial sentence for a specific reason, but this should be consistent with other instructions.

Reply:
I thought we were dropping "replica" and using "instance" instead?

Reply:
Yeah, I just didn't since I made this suggestion before noticing Andy's comment. Edited.